services/src/datasets/external/gfbio.rs

```diff
@@ -212,6 +212,8 @@ impl ExternalDatasetProvider for GfbioDataProvider {
                         .filter(|(_, name)| name.starts_with("/DataSets/DataSet/Units/Unit/"))
                         .map(|(_, name)| (name.clone(), FeatureDataType::Text))
                         .collect(),
+                    time: None,
+                    bbox: None,
```

Comment:
Maybe add some todos. I think we have no time/resources to adapt the source, but actually we would need to get the values.

Reply:
66619e218c2104c8577669937023a7354353f531